src/javascript/binary_japan/symbols.js

```diff
@@ -0,0 +1,20 @@
+if(typeof is_japan === 'function'){
+	Symbols._details = Symbols.details.bind({});
+
+	Object.defineProperties(Symbols,{
+		details:{
+			value:function(data){
+				var active_symbols = [];
+
+				data.active_symbols.forEach(function(symbol){
```

Comment:
We've used the forEach + push in new array a lot in the past.
If you are not familiar, read on the map() method of Array (it is supported in ES5 thus all browsers we support)